File: KarateKlub/KarateKlub.Api/Models/ClanarineIzmireneClanarineClanaPregledVM.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace KarateKlub.Api.Models
{

    public class ClanarineIzmireneClanarineClanaPregledVM
    {
        public class Row
        {
            public int id { get; set; }
            public string naziv { get; set; }
            public string mjestoUplate { get; set; }
            public string iznos { get; set; }
            public DateTime? datumUplate { get; set; }
            public DateTime datumOd { get; set; }
            public DateTime datumDo { get; set; }
            public string brojPriznanice { get; set; }
        }
        public List<Row> rows { get; set; }
    }
}

File: Android aplikacija/KarateKlub/app/src/main/java/com/example/x/karateklub/area_clan/model/ClanarineIzmireneClanarineClanaPregledVM.java
package com.example.x.karateklub.area_clan.model;

import java.io.Serializable;
import java.util.Date;
import java.util.List;

public class ClanarineIzmireneClanarineClanaPregledVM implements Serializable {
    public static class Row implements Serializable{
        public int id;
        public String naziv;
        public String mjestoUplate;
        public String iznos;
        public Date datumUplate;
        public Date datumOd;
        public Date datumDo;
        public String brojPriznanice;

    }



        public List<Row> rows;

}
